src/main/java/org/folio/util/TokenCookieParser.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.util;
+
+import org.folio.rest.impl.LoginAPI;
+
+public class TokenCookieParser {
+  public String accessToken;
+  public String refreshToken;
+
+  public TokenCookieParser(String cookieHeader) {
+    accessToken = getCookieValue(cookieHeader, LoginAPI.ACCESS_TOKEN);
+    refreshToken = getCookieValue(cookieHeader, LoginAPI.REFRESH_TOKEN);
+  }
+
+  private String getCookieValue(String cookieHeader, String key) {
+    String[] cookies = cookieHeader.split(";");
+    for (String c : cookies) {
+      String[] cookeNameValue = c.trim().split("=", 2);
+      String cookieName = cookeNameValue[0].trim();
+      String cookieValue = cookeNameValue[1].trim();
+      if (cookieName.equals(key)) {
+        return cookieValue;
```

Comment:
https://datatracker.ietf.org/doc/html/rfc6265#section-4.2.2 :
> servers SHOULD NOT rely upon the serialization order.  In particular,
   if the Cookie header contains two cookies with the same name (e.g.,
   that were set with different Path or Domain attributes), servers
   SHOULD NOT rely upon the order in which these cookies appear in the
   header.

If a cookieName appears more than once it should be rejected with an Exception.

Reply:
Ok sure. Added some code for this.